katuma.yml

```diff
@@ -14,3 +14,14 @@
         log_path: "{{ shared_path }}/log"
         pids_path: "{{ shared_path }}/pids"
         ofn_route_override: "/admin/reports/order_cycle_management"
+
+    - role: jdauphant.nginx
+      nginx_sites:
+        katuma_redirect:
+          - listen 443 ssl http2
+          - listen [::]:443 ssl http2
+          - server_name staging.katuma.org
+          - return 301 https://appstaging.katuma.org$request_uri
```

Comment:
appstaging? :eyes:

Reply:
don't ask, I have nasty things to try things out :trollface: